nixos/modules/services/networking/dnsmasq.nix

```diff
@@ -62,6 +62,15 @@ in
         '';
       };
```

Comment:
I would update the option description here to mention this will by default resolve upstream DNS services and if that's not wanted, to set `useResolvConfUpstreams` to `false`.